File: netDxf/IO/DxfWriter.StoredSunStudy.cs
// Copyright (c) netDxf contributors. Licensed under the MIT License.
using netDxf.Objects;

namespace netDxf.IO
{
    internal sealed partial class DxfWriter
    {
        private bool WriteStoredSunStudyPayload(DxfDatabaseObject item)
        {
            if (!(item is DxfStoredSunStudy study)) return false;
            foreach (DxfTag tag in study.Payload) this.WriteDatabaseTag(tag, false);
            return true;
        }
    }
}

File: javascript/netDxf/IO/DxfWriter.StoredSunStudy.js
// Copyright (c) netDxf contributors. MIT License; see package LICENSE.
import { DxfStoredSunStudy } from '../Objects/DxfStoredSunStudy.js';
export function WriteStoredSunStudyPayload(chunk,version,item){
  if(!(item instanceof DxfStoredSunStudy))return false;
  for(const tag of item.Payload)chunk.Write(tag.Code,tag.Value);return true;
}
